scripts/start_civ.sh

```diff
@@ -463,7 +465,10 @@ function is_usb_dev_udc()
 
 function set_pt_usb() {
     local d
-    local USB_PCI=$(lspci -D |grep -m1 "USB controller" | grep -o "....:..:..\..")
+
+    for i in $XHCI_CONTROLLERS_VENDORID; do
+        USB_PCI="$USB_PCI $(lspci -nn -D |grep  "USB controller" | grep $i | awk '{print $1}')"
```

Comment:
As discussion with Sundar. To add a new option might OK, like --passthrough-usb-tgl-cml or --passthrough-usb-00-14-0 or some other name. Do not modify set_pt_usb(), add a new function instead.